### PR TITLE
Fixed broadcaster and biometric, security problems

### PR DESCRIPTION
**Pull request by hitori666**
Fixed the issue of Broadcast Receiver and biometric, this was reported used mobsf.
is necesary test to check the notifications.

### CODE REVIEW

**Comment by nicolasburtey** [comment]
what are the binary files for?

**Comment by hitori666** [comment]
sorry was autogenerated, is not important, are empthy files, you can ignore this 2 .bin

**Comment by nicolasburtey** [comment]
can you remove/rebase? we don't want binary file in our repo

**Comment by hitori666** [comment]
Done, bin files was deleted

**Comment by nicolasburtey** [comment]
rebase still needed to clean up github history

independently on this I don't understand what security problem this PR fixes. more context would be useful

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,7 +10,25 @@
     <uses-permission android:name="android.permission.USE_BIOMETRIC" />
     <uses-permission android:name="android.permission.USE_FINGERPRINT" />
 
-    <!-- TODO android:roundIcon="@mipmap/ic_launcher_round" -->
+	<!-- reciber problem security" -->
```
Comment (dolcalmi) [inline android/app/src/main/AndroidManifest.xml:13]: please remove this comment

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -10,7 +10,25 @@
...
+		</intent-filter>
+	</receiver>
+
+<!-- biometric security problem fix -->
+	<activity android:name="androidx.biometric.DeviceCredentialHandlerActivity"
+            android:exported="false"
+            tools:replace="android:exported">
+	</activity>
+	
+	<!-- TODO android:roundIcon="@mipmap/ic_launcher_round" -->
```
Comment (dolcalmi) [inline android/app/src/main/AndroidManifest.xml:31]: same here

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -10,7 +10,25 @@
...
+			<action android:name="com.google.android.c2dm.intent.RECEIVE"/>
+		</intent-filter>
+	</receiver>
+	<receiver android:exported="false" android:name="com.google.firebase.iid.FirebaseInstanceIdReceiver" android:permission="com.google.android.c2dm.permission.SEND">
+		<intent-filter>
+			<action android:name="com.google.android.c2dm.intent.RECEIVE"/>
+		</intent-filter>
+	</receiver>
+
+<!-- biometric security problem fix -->
```
Comment (dolcalmi) [inline android/app/src/main/AndroidManifest.xml:25]: same here

**Review by dolcalmi — CHANGES_REQUESTED** [review]

**Review by hitori666 — COMMENTED** [review]
Comments removed

**Comment by hitori666** [comment]
Rebase completed, this fixed problems that mobsf detect in the apk, one of they are Broadcast Receiver and ctivity (androidx.biometric.DeviceCredentialHandlerActivity) is not Protected. [android:exported=true], this gonna help to improve the score of the app, now we have a Average CVSS Score: 6.4, whit this change, is possible to have a better score. 
this is important for the security analisis of the app

File: android/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -10,7 +10,23 @@
     <uses-permission android:name="android.permission.USE_BIOMETRIC" />
     <uses-permission android:name="android.permission.USE_FINGERPRINT" />
 
-    <!-- TODO android:roundIcon="@mipmap/ic_launcher_round" -->
+	
+	<receiver android:exported="false" android:name="io.invertase.firebase.messaging.ReactNativeFirebaseMessagi ngReceiver" android:permission="com.google.android.c2dm.permission.SEND">
```
Comment (jrbuhl93) [inline android/app/src/main/AndroidManifest.xml:14]: `ReactNativeFirebaseMessagi ngReceiver" ` shouldn't have a space

Comment (hitori666) [reply android/app/src/main/AndroidManifest.xml:14]: Done

**Comment by daviroo** [comment]
This looks pretty stale to me.  Going to close it.

**Pull request closed without merging**